components/date-picker/index.en-US.md

```diff
@@ -0,0 +1,74 @@
+---
+category: Components
+type: Form Control
+english: DatePicker
+---
+
+To select/input a date.
+
+## When To Use
+
+By clicking the input box, you can select a date from a popup calendar.
+
+## API
+
+### DatePicker
+
+```html
+<DatePicker defaultValue="2015-01-01" />
+```
+
+> Warning: `Datepicker` is renamed to `DatePicker` after `0.11`.
+
+
+| Property         | Description           | Type     | Default       |
+|--------------|----------------|----------|--------------|
+| value        | to set date           | String/Date   | -           |
+| defaultValue | to set default date        | String/Date   | -           |
+| format       | to set the date format, refer to [GregorianCalendarFormat](https://github.com/yiminghe/gregorian-calendar-format) | String   | "yyyy-MM-dd" |
+| disabledDate | to specify the date that cannot be selected | function | -           |
+| onChange     | a callback function, can be executed when the selected time is changing | function(date, dateString) | -           |
+| disabled     | determine whether the DatePicker is disabled           | Boolean     | false        |
+| style        | to customize the style of the input box     | Object     | {}   |
+| popupStyle   | to customize the style of the popup calendar   | Object     | {}   |
+| size         | determine the size of the input box, the height of `large` and `small`, were 32px and 22px respectively, while default size is 28px | String   | -  |
```

Comment:
| size         | determine the size of the input box, the height of `large` and `small`, **are** 32px and 22px respectively, while default size is 28px | String   | -  |